pkg/operator/controller/status/controller_test.go

```diff
@@ -8,14 +8,10 @@ import (
 
 	configv1 "github.com/openshift/api/config/v1"
 
-	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-func TestComputeOperatorStatusConditions(t *testing.T) {
-	type conditions struct {
-		degraded, progressing, available bool
-	}
+func TestComputeOperatorProgressingCondition(t *testing.T) {
```

Comment:
So we're losing unit-test coverage of `computeOperatorAvailableCondition` and `computeOperatorDegradedCondition`, but I suppose they are sufficiently covered by E2E tests.

Reply:
They currently seem simple enough that unit test coverage would be more code than it's worth given e2e, IMO.

If we start introducing additional inputs to the formulas, unit tests may become useful...